openslides/config/views.py

```diff
@@ -100,3 +101,22 @@ def form_valid(self, form):
             config[key] = form.cleaned_data[key]
         messages.success(self.request, _('%s settings successfully saved.') % _(self.config_collection.title))
         return super(ConfigView, self).form_valid(form)
+
+
+class ConfigViewSet(viewsets.ViewSet):
```

Comment:
I think this is a more general problem. It could be, that the config-app can not be accessable via rest, but though another api which is more suable for config values. 

If we add the config now, it will be harder to change the client later. So I would suggest, you remove the config from the rest api for now.